components/eventing-controller/controllers/backend/reconciler.go

```diff
@@ -103,6 +113,12 @@ func NewReconciler(ctx context.Context, natsSubMgr subscriptionmanager.Manager,
 // +kubebuilder:rbac:groups="",resources=pods,verbs=get;list;watch;delete
 
 func (r *Reconciler) Reconcile(ctx context.Context, _ ctrl.Request) (ctrl.Result, error) {
+	if r.natsConfig.EnableNewCRDVersion {
+		if err := r.updateMutatingValidatingWebhookWithCABundle(ctx); err != nil {
```

Comment:
Exactly! This was the hack I had to do so that we could attach the CABundle to the mutating and validating webhook configuration. 
A global cert manager would be perfect for us and we can avoid doing this.